docs/docs/concepts/lcel.mdx

```diff
@@ -0,0 +1,106 @@
+# LangChain Expression Language (LCEL)
```

Comment:
This is WIP -- I want to reduce the Why LCEL section so it's faster to read tthrough

will expand later more detail about composition primitives etc